apps/backend/src/events/events.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Body, Controller, Delete, Get, Param, Patch, Post } from '@nestjs/common';
+import { CreateEventDto } from './dto/create-event.dto';
+import { UpdateEventDto } from './dto/update-event.dto';
+import { EventsService } from './events.service';
+
+@Controller('events')
+export class EventsController {
+  constructor(private readonly eventsService: EventsService) {}
+
+  @Post()
+  create(@Body() createEventDto: CreateEventDto) {
+    return this.eventsService.create(createEventDto);
+  }
+
+  @Get()
+  findAll() {
+    return this.eventsService.findAll();
+  }
+
+  @Get(':id')
+  findOne(@Param('id') id: string) {
```

Comment:
we should validate the id here. In the course, we used ParseIntPipe, but that's not going to work here since we don't use number based ID.s Instead take a look at ParseUUIDPipe

[https://docs.nestjs.com/pipes](https://docs.nestjs.com/pipes)